### ISSUE TITLE
Component doesn't support compilation with `--no-fetch-dependencies`

### ISSUE THREAD

**Issue by simu** [projectsyn/component-topolvm#13]
The topolvm component fails to compile with `commodore catalog compile ${CLUSTER_ID} --local -F`.

## Steps to Reproduce the Problem

  1. Compile catalog normally with `commodore catalog compile ${CLUSTER_ID}`
  1. Compile catalog again in local mode with `--no-fetch-dependencies` (`-F`): `commodore catalog compile ${CLUSTER_ID} --local -F`

## Actual Behavior

Local compile with `-F` fails with 

```
Compile error: topolvm/helmcharts/topolvm/topolvm-chart-v3.1.1/templates/lvmd/configmap.yaml for target: topolvm not found in search_paths: [PosixPath('/home/simon/work/ocp4/c-old-fire-5788/vendor'), PosixPath('/home/simon/work/ocp4/c-old-fire-5788'), PosixPath('/home/simon/.local/lib/python3.8/site-packages/commodore'), '/tmp/tmp8bk9woxw.kapitan']
```

Note that the Kapitan `remove` input type ([implementation](https://github.com/kapicorp/kapitan/blob/1cd429102dda0a7cee6525a2e61c2b88f50ab292/kapitan/inputs/remove.py#L22)) doesn't support parameter `ignore_missing` which is provided in https://github.com/projectsyn/component-topolvm/blob/9fdcf6e32d81ff8fef0018d73948fb13179d71ae/class/topolvm.yml#L35

## Expected Behavior

Compilation is successful

**Comment by DebakelOrakel** [comment]
I can't reproduce this issue. Followed your steps:
* commodore compile {cluster_id}
* commodore compile {cluster_id} --local --no-fetch-dependencies
No error

Deleted all the folders from commodore, and did above commands again -> No error.

**Comment by simu** [comment]
Yeah, I think this issue has been resolved with Commodore 0.11 / Kapitan 0.30.
